From de5f45b47ba3b842063b7d4802db672f205e1ef7 Mon Sep 17 00:00:00 2001
From: Steven Adger <adgitate@gmail.com>
Date: Sun, 11 Feb 2018 15:29:31 -0700
Subject: [PATCH 128/249] klte variants: Fix TWRP codename/link

The official TWRP page for Samsung Galaxy S5
(https://twrp.me/samsung/samsunggalaxys5qualcomm.html) links:

-G900I and G900P to "klte",
-G900K, G900L and G900S to "klteskt",
-G9006W and G9008W to "kltechn"

Change-Id: Ifb7a7803080f0d77d2432707571895e69c2a3743
---
 _data/devices/kltechnduo.yml | 1 +
 _data/devices/kltedv.yml     | 1 +
 _data/devices/kltekor.yml    | 1 +
 3 files changed, 3 insertions(+)

diff --git a/_data/devices/kltechnduo.yml b/_data/devices/kltechnduo.yml
index 64fd5f3..470560c 100644
--- a/_data/devices/kltechnduo.yml
+++ b/_data/devices/kltechnduo.yml
@@ -11,6 +11,7 @@ cpu: Krait 400
 cpu_cores: '4'
 cpu_freq: 2.5 GHz
 current_branch: 14.1
+custom_twrp_codename: kltechn
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 330
 has_recovery_partition: true
diff --git a/_data/devices/kltedv.yml b/_data/devices/kltedv.yml
index e81189d..8e2d4dd 100644
--- a/_data/devices/kltedv.yml
+++ b/_data/devices/kltedv.yml
@@ -11,6 +11,7 @@ cpu: Krait 400
 cpu_cores: '4'
 cpu_freq: 2.5 GHz
 current_branch: 14.1
+custom_twrp_codename: klte
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 330
 has_recovery_partition: true
diff --git a/_data/devices/kltekor.yml b/_data/devices/kltekor.yml
index 9216e27..e7130e3 100644
--- a/_data/devices/kltekor.yml
+++ b/_data/devices/kltekor.yml
@@ -11,6 +11,7 @@ cpu: Krait 400
 cpu_cores: '4'
 cpu_freq: 2.5 GHz
 current_branch: 14.1
+custom_twrp_codename: klteskt
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 330
 has_recovery_partition: true
-- 
2.17.1

